crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net.rs

```diff
@@ -415,7 +418,10 @@ impl MetaNet {
             while let Some(evt) = ep.next().await {
                 match evt {
                     Tx2EpEvent::OutgoingConnection(Tx2EpConnection { con, url }) => {
-                        if evt_send
+                        if !matches!(
+                            host.is_blocked(con.clone().into(), Timestamp::now()).await,
+                            Ok(false)
+                        ) || evt_send
```

Comment:
So, if `is_blocked` returns anything other than `Ok(false)` we break out of the loop processing all communications with all peers.

I think instead we need to close this one specific connection.

Also, does it not work to do it on the receive side of this send channel? Then it wouldn't have to be differentiated between tx2 and tx5.

Reply:
here: [https://github.com/holochain/holochain/blob/988959365281e45aba63864b85ca0fc1e35eb8ac/crates/kitsune_p2p/kitsune_p2p/src/spawn/actor.rs#L271](https://github.com/holochain/holochain/blob/988959365281e45aba63864b85ca0fc1e35eb8ac/crates/kitsune_p2p/kitsune_p2p/src/spawn/actor.rs#L271)

I know I pointed you to this spot earlier. Sorry for not thinking of this better spot sooner.